crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -198,7 +249,7 @@ mod tests {
         assert_eq!(response.status(), StatusCode::OK);
         assert_eq!(
             response.headers().get(&VERSION_HEADER),
-            Some(&HeaderValue::from_static(RPC_VERSION_FULL)),
+            Some(&HeaderValue::from_str(test_version().to_string().as_str()).unwrap())
```

Comment:
This assertion might look cleaner if you convert the header value you get from the response to a string rather than convert the version into a header value.